docs/provisioner.md

```diff
@@ -61,3 +65,9 @@ endpoint `/metrics` on the listening port (default 8080).
 | local_volume_provisioner_proctable_running                    | Gauge       |                                                                                                                                                                                    |
 | local_volume_provisioner_proctable_failed                     | Gauge       |                                                                                                                                                                                    |
 | local_volume_provisioner_proctable_succeeded                  | Gauge       |                                                                                                                                                                                    |
+
+### Readiness
+
+The readiness state is exposed on the path `/ready`.
+
+The state become ready when discovered local volumes are successfully created.
```

Comment:
one more nit clarification: on the initial pass

Reply:
Would it be worth calling out that if there were no disks found on the initial pass, then it will also be ready?

Reply:
@msau42 it's not on the initial pass only. For example if the RBAC is removed the pv creation will fail so the pod will become notReady.

This is what I had in mind, would you like me to change this to become ready on the initial pass only ?

Reply:
Hm ok I think the rbac removal afterwards is a valid scenario of it becoming not ready.  The main scenario I was thinking of was:

1. Start provisioner before any disks were added/mounted
2. Provisioner sets ready = true because there were no disks to discover
3. After disks were added/mounted, now there are issues, and readiness gets set to false.

Is it ok for the provisioner to have been ready in step 2?  I think it's fine (and I don't see an easy way to really detect this scenario), but do you think it's worth clarifying that it might be ready even if there were no disks and/or no work to do (ie all the disks were already discovered)?

Reply:
@msau42 Agreed, I'm going to add some information about this in the docs 👍